File: SDKs/netboxdemo.com/2.4/go/pkg/models/operations/circuitscircuitterminationscreate.go
package operations

import (
	"openapi/pkg/models/shared"
)

type CircuitsCircuitTerminationsCreateRequest struct {
	Request shared.WritableCircuitTerminationInput `request:"mediaType=application/json"`
}

type CircuitsCircuitTerminationsCreateResponse struct {
	CircuitTermination *shared.CircuitTermination
	ContentType        string
	StatusCode         int64
}

File: SDKs/netboxdemo.com/2.4/typescript/src/sdk/models/operations/circuitscircuitterminationscreate.ts
import { Metadata, SpeakeasyBase } from "../../../internal/utils/utils";
import * as shared from "../shared";


export class CircuitsCircuitTerminationsCreateRequest extends SpeakeasyBase {
  @Metadata({ data: "request, media_type=application/json" })
  request: shared.WritableCircuitTermination;
}


export class CircuitsCircuitTerminationsCreateResponse extends SpeakeasyBase {
  @Metadata()
  circuitTermination?: shared.CircuitTermination;

  @Metadata()
  contentType: string;

  @Metadata()
  statusCode: number;
}
